cmd/watch-main.go

```diff
@@ -97,8 +97,17 @@ func checkWatchSyntax(ctx *cli.Context) {
 // watchMessage container to hold one event notification
 type watchMessage struct {
 	Status string `json:"status"`
-	Event  Event  `json:"events"`
-	Source Source `json:"source,omitempty"`
+	Event  struct {
+		Time string    `json:"time"`
+		Size int64     `json:"size"`
+		Path string    `json:"path"`
+		Type EventType `json:"type"`
+	} `json:"events"`
+	Source struct {
+		IP        string `json:"ip,omitempty"`
+		Port      string `json:"port,omitempty"`
+		UserAgent string `json:"userAgent,omitempty"`
+	} `json:"source,omitempty"`
```

Comment:
Status is not useful.

EvenInfo can be used for printing and `watchMessage` can be eliminated completely.